bugbug/bugzilla.py

```diff
@@ -93,50 +94,84 @@ def bughandler(bug):
     return all_ids
 
 
-def get(ids_or_query):
-    new_bugs = {}
+def get(bug_ids: Union[List[int], Set[int]]):
```

Comment:
You just need something that is iterable, so use `Iterable[int]` instead of this `Union`

Reply:
Updated the source code by replacing `Union` with `Iterable`